src/SensioLabs/Connect/Api/Entity/Root.php

```diff
@@ -31,11 +28,17 @@ protected function configure()
         ;
     }
 
+    /**
+     * @deprecated since Connect 4.2.4, will be removed in 5.0.
```

Comment:
should trigger a runtime notice too (same in other methods)

Reply:
Fixed.